.github/workflows/dusk_ci.yml

```diff
@@ -0,0 +1,82 @@
+on: [pull_request]
+
+name: Continuous integration
+
+jobs:
+  analyze:
+    name: Dusk Analyzer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: --git https://github.com/dusk-network/cargo-dusk-analyzer
+      - uses: actions-rs/cargo@v1
+        with:
+          command: dusk-analyzer
+
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  test_stable:
+    name: Stable tests
+    runs-on: [ubuntu-latest, macos-latest]
```

Comment:
Limit to `ubuntu-latest`, like we have for the rest of the jobs and repos' actions. Not quite sure why you added macos-latest, here.

Also, it seems to cause issue and the jobs is not running: 
```
No runner matching the specified labels was found: ubuntu-latest, macos-latest
```

Reply:
Yep, I wanted to try it but I'd need to look at it better.